### DISCUSSION TITLE
Add the Clip-path Converter to the tools list

### DISCUSSION THREAD

**Discussion by Panquesito7** [devicons/devicon#1674] [Ideas]
Recently, I found this amazing tool to convert SVG clip paths to relative SVG code.
This can help a lot if you do not want to use clip paths in your SVGs (and, for example, Devicon fonts).

Link to the project: https://yoksel.github.io/relative-clip-path/
GitHub link (in case you'd like to contribute): https://github.com/yoksel/relative-clip-path/

What do you think? Should we add this to the recommended resources list?

**Comment by Snailedlt** [comment]
Looks useful. Feel free to add it :)

**Reply by Panquesito7** [reply]
Added. :)
